setup.py

```diff
@@ -20,14 +20,20 @@ def get_requirements():
 
     requirements = {}
     for req in meta["requirements"]["run"]:
-        requirement = Requirement(req)
+        try:
+            requirement = Requirement(req)
+        except InvalidRequirement as e:
```

Comment:
We can't really use `conda` here, so we have to resort to catching the error instead.